libkbfs/md_journal.go

```diff
@@ -1178,7 +1226,11 @@ func (j *mdJournal) clear(
 	// Garbage-collect the old branch entries.  TODO: we'll eventually
 	// need a sweeper to clean up entries left behind if we crash
 	// here.
+	isLocalSquash := bid == LocalSquashBranchID
```

Comment:
There's also a check above that `head.BID() == j.branchID`, which you'll need to fix up.